Source/WebKit/NetworkProcess/webrtc/NetworkMDNSRegister.cpp

```diff
@@ -98,24 +91,23 @@ void NetworkMDNSRegister::closeAndForgetService(DNSServiceRef service)
     });
 }
 
-static void registerMDNSNameCallback(DNSServiceRef service, DNSRecordRef record, DNSServiceFlags, DNSServiceErrorType errorCode, void *context)
+static void registerMDNSNameCallback(DNSServiceRef service, DNSRecordRef record, DNSServiceFlags, DNSServiceErrorType errorCode, void* context)
 {
-    auto request = pendingRegistrationRequests().take(reinterpret_cast<uintptr_t>(context));
-    if (!request)
-        return;
+    auto* request = reinterpret_cast<PendingRegistrationRequest*>(context);
```

Comment:
Seems we could construct a std::unique_ptr for request from the raw pointer here, to avoid manual `delete` calls in several code paths in this function.